clang/test/Driver/mcmodel.c

```diff
@@ -44,3 +44,5 @@
 // ERR-AARCH64_32: error: unsupported argument 'small' to option '-mcmodel=' for target 'aarch64_32-unknown-linux'
 
 // ERR-LOONGARCH64-PLT-EXTREME: error: invalid argument '-mcmodel=extreme' not allowed with '-fplt'
+
+// CHECK: error: unsupported argument 'tiny' to option '-mcmodel=' for target '{{.*}}'
```

Comment:
We have nothing with a check prefix of just `CHECK` so this will never actually be tested. Also, this is the same message as `ERR-TINY` so we wouldn't need this line anyway.